search-app/src/cmr/search/site/routes.clj

```diff
@@ -23,6 +23,12 @@
         (GET "/"
              {context :request-context}
              (pages/home context))
+        (GET "/sitemap.xml"
+             {context :request-context}
+             (pages/sitemap-master context))
+        (GET "/site/sitemap.xml"
```

Comment:
`sitemap.xml` is the master -- it uses the sitemap "index" schema (as opposed to the regular sitemap schema). It is my understanding that the sitemap index schema only supports collections of sitemaps, not collections of content locations.